torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -167,6 +172,66 @@ void fuseTransposeIntoGemm(std::shared_ptr<Graph>& graph) {
   }
 }
 
+void pushPackingPastRnn(std::shared_ptr<Graph>& graph) {
```

Comment:
Want a comment here. The discussion from the PR is a good start for the text here.

Reply:
++